From 699d9584d4332a73d94de8b0ad0f3cea77c81f55 Mon Sep 17 00:00:00 2001
From: SeongJae Park <sj@kernel.org>
Date: Mon, 29 Apr 2024 17:16:56 -0700
Subject: [PATCH] Docs/mm/damon/maintainer-profile: mention HacKerMaiL for
 beginners

DAMON community uses the mailing list based workflow.  That's not very
easy for every beginner.  DAMON maintainer is also developing a tool for
the mailing list based workflow for himself.  The maintainer would be
able to provide a better support if the request is for the tool and came
from a potential DAMON contributor.  Mention it as one of recommended
tools for such begineers.

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 Documentation/mm/damon/maintainer-profile.rst | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/Documentation/mm/damon/maintainer-profile.rst b/Documentation/mm/damon/maintainer-profile.rst
index 8213cf61d38a..2d6adfa03326 100644
--- a/Documentation/mm/damon/maintainer-profile.rst
+++ b/Documentation/mm/damon/maintainer-profile.rst
@@ -10,6 +10,11 @@ The mailing lists for the subsystem are damon@lists.linux.dev and
 linux-mm@kvack.org.  Patches should be made against the mm-unstable tree [1]_
 whenever possible and posted to the mailing lists.
 
+For beginners who are not familiar with the mailing lists based workflow, there
+are multiple tools.  HacKerMaiL (hkml) [8]_ is one of such tools that DAMON
+maintainer is developing for such beginners to DAMON.  Please feel free to try
+the tool and ask help to the maintainer.
+
 SCM Trees
 ---------
 
@@ -61,3 +66,4 @@ of sending a patch.
 .. [5] https://github.com/awslabs/damon-tests/blob/master/corr/tests/kunit.sh
 .. [6] https://github.com/awslabs/damon-tests/tree/master/corr
 .. [7] https://github.com/awslabs/damon-tests/tree/master/perf
+.. [8] https://github.com/sjp38/hackermail
-- 
2.39.2

